app/assets/javascripts/admin/addon/templates/modal/admin-penalize-user.hbs

```diff
@@ -0,0 +1,63 @@
+<DModalBody @title={{this.modalTitle}}>
+  <ConditionalLoadingSpinner @condition={{this.loadingUser}}>
+    {{#if this.errorMessage}}
+      <div class="alert alert-error">{{this.errorMessage}}</div>
+    {{/if}}
+
+    {{#if this.canPenalize}}
+      <div class="penalty-duration-controls">
+        {{#if (eq this.penaltyType "suspend")}}
+          <FutureDateInput @class="suspend-until"
+                           @label="admin.user.suspend_duration"
+                           @clearable={{false}}
+                           @input={{this.penalizeUntil}}
+                           @onChangeInput={{action (mut this.penalizeUntil)}} />
+        {{else if (eq this.penaltyType "silence")}}
+          <FutureDateInput @class="silence-until"
+                           @label="admin.user.silence_duration"
+                           @clearable={{false}}
+                           @input={{this.penalizeUntil}}
+                           @onChangeInput={{action (mut this.penalizeUntil)}} />
```

Comment:
I think those two `@onChangeInput`s can be simply `{{mut this.penalizeUntil}}`

Reply:
I tested it, but it did not work. :(

Reply:
Hm, interesting. Will take a look after it's merged!